patients/repo.go

```diff
@@ -1317,17 +1321,20 @@ func (r *repository) TideReport(ctx context.Context, clinicId string, params Tid
 		}
 
 		opts := options.Find()
-		opts.SetLimit(int64(limit))
+		opts.SetLimit(TideReportPatientLimit)
```

Comment:
Mostly curiousity question: the original line had `int64` cast, as does the latter use on line 1366. Does it make a difference either here or there?